kcidb/monitor/output.py

```diff
@@ -1,5 +1,5 @@
 """Kernel CI reporting - monitor - output"""
-
```

Comment:
Could you please not remove this blank line? They're here in every module, as a convention, and they help readability a little.